gendst/data/positions.go

```diff
@@ -97,7 +97,7 @@ type T /*Start*/ func /*Func*/ (a int) /*Params*/ (b int) /*End*/
 
```

Comment:
In here we need something to show FuncType.TypeParams, like this:

```go
// FuncType(0)
type T1 /*Start*/ func /*Func*/ (a int) /*Params*/ (b int) /*End*/

// FuncType(1)
func F1 /*Start*/ [P any] /*TypeParams*/ (p P) /*Params*/ P/*End*/ { return p }
```

Reply:
No reason to be sorry... better to think these things out and take time than rush.

Reply:
Example added!